doc/tutorial/deploying.rst

```diff
@@ -189,7 +189,7 @@ contents:
        steps:
        - uses: actions/checkout@v4
        - name: Build HTML
-         uses: ammaraskar/sphinx-action@master
+         uses: ammaraskar/sphinx-action@8.0.2
        - name: Upload artifacts
          uses: actions/upload-artifact@v4
          with:
```

Comment:
Commenting as near to the relevant line as the GitHub web interface will allow me:

Perhaps we should consider updating the `peaceiris/actions-gh-pages` action at the same time?  The [differences between `v3` and `v4` of that plugin](https://github.com/peaceiris/actions-gh-pages/compare/v3...v4) appear to be minimal, and are largely dependency-updates.